rocAL_pybind/rocal_pybind.cpp

```diff
@@ -141,6 +141,8 @@ std::unordered_map<int, std::string> rocalToPybindOutputDtype = {
     {1, "float16"},
     {2, "uint8"},
     {3, "int8"},
+    {4, "uint32"},
```

Comment:
In Rocal_api_types.h there is not uint32 or int32 please check

Reply:
We are using the VX TYPE INT32 in tensor.cpp . Added in the rocal_api_types.h too